spec/bulk_processor/back_end/dynosaur/split_csv_task_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'bulk_processor/back_end/dynosaur/split_csv_task'
+
+describe 'bulk_processor namespace' do
+  describe 'bulk_processor:split' do
+    let(:performer) { instance_double(BulkProcessor::SplitCSV, perform: true) }
+    let(:rake_cmd) do
+      'bulk_processor:split[MockCSVProcessor,foo=bar,file.csv,2]'
+    end
+
+    before(:all) { Rake::Task.define_task(:environment) }
```

Comment:
So for someone to use this gem, they're going to need a `rake environment` task. That might be a nice requirement to remove in the future.